### PR TITLE
Update .travis.yml and fix the CI build

### PR DESCRIPTION
**Pull request by josePhoenix**
I understand there is a new way to write `.travis.yml` files in Astropy's package template, but I haven't yet gotten it to work. This fixes the file we had so at least the build runs.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -26,59 +33,43 @@ matrix:
...
-        # - python: 3.3
-        #   env: ASTROPY_VERSION=development SETUP_CMD='test'
 
-        # Try older numpy version
-        # Use (temporarily?) the dev astropy to work around the issue in 
-        # https://github.com/mperrin/poppy/issues/87 
-        # https://github.com/astropy/astropy-helpers/issues/73#issuecomment-72558556
+        # Test one previous version of NumPy
         - python: 2.7
           env: NUMPY_VERSION=OLDER_NUMPY_VERSION ASTROPY_VERSION=stable SETUP_CMD='test'
```
Comment (mperrin) [inline .travis.yml]: I think this needs a $ before the OLDER_NUMPY_VERSION

File: .travis.yml [hunk 2]
```diff
@@ -26,59 +33,43 @@ matrix:
         # Do a coverage test in Python 3.
         - python: 3.5
           env: SETUP_CMD='test --coverage'
-        # Check for sphinx doc build warnings - we do this first because it
-        # may run for a long time. 
-        # Update: ignore warnings, just check for errors
+        # Check for sphinx doc build errors
         - python: 3.5
           env: SETUP_CMD='build_sphinx'
```
Comment (mperrin) [inline .travis.yml]: I thought we might as well switch the docs build to 3.5, but I see we are getting some bogus distutils error on that now. Grrr.

**Comment by josePhoenix** [comment]
Haha, well it was working before the merge! 😉 I'll take a look.

**Comment by mperrin** [comment]
woohoo, and you managed to fix the coverage tests build too!

**Comment by josePhoenix** [comment]
I thought you did that!

**Comment by mperrin** [comment]
I did nothing specifically related to coveralls! Maybe it was broken before by the outdated version of numpy or something random like that?

**Comment by mperrin** [comment]
now I want to write a bunch of unit tests for easy stuff to boost the coverage percentage. the incentives are working....

**Pull request merged**